addons/payment/tests/common.py

```diff
@@ -96,6 +96,7 @@ def _get_payment_method_information(self):
         cls.acquirer = cls.dummy_acquirer
         cls.amount = 1111.11
         cls.company = cls.env.company
+        cls.company_id = cls.company.id
```

Comment:
why this change ?

Reply:
`env.company` is a property and some IDEs have issues with `id` being read from `self.company` in tests, so I extracted the value from `cls.company` here.